Source/devices/oni/AcqBoardONI.cpp

```diff
@@ -201,8 +199,8 @@ void AcqBoardONI::updateCustomStreams (OwnedArray<DataStream>& otherStreams, Own
         ContinuousChannel::AUX,
         "MEM",
         "Hardware buffer usage",
-        "rhythm-fpga-device.continuous.mem",
-        1.0f, //some scaling so it does not overflow too easily
+        "acq-board.memory.continuous",
```

Comment:
I just thought, maybe call this continuous.percentage ?
just to be clear on what this entails
Also, can you add the unit as percentage?